src/App.js

```diff
@@ -9,6 +9,10 @@ function App() {
       <h2>BookStore</h2>
       <BooksList />
       <BooksForm />
+      <br />
+      <table>
+        <BooksList />
+      </table>
```

Comment:
- [ ] [REQUIRED] Please make sure that the list of books is displayed only once.